src/routes/home/components/spinner/style.css

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Load Awesome v1.1.0 (http://github.danielcardoso.net/load-awesome/)
+ * Copyright 2015 Daniel Cardoso <@DanielCardoso>
+ * Licensed under MIT
+ */
+.spinner {
+    --height: 128px;
```

Comment:
This won't work in most of the browsers, maybe preact-cli has something to do it, or we could just remove the variables 😂